urns/urns.go

```diff
@@ -254,7 +272,7 @@ func (u URN) Resolve(key string) interface{} {
 func (u URN) Default() interface{} { return u }
 
 // String returns the string representation of this URN
-func (u URN) String() string { return string(u.Path()) }
+func (u URN) String() string { return string(u) }
```

Comment:
Sure seems like the string value for a URN should be the full representation for the URN. This kind of maps to the values for locations and times being fully representative and something like `format_urn` being responsible for creating user friendly versions.

Reply:
Yeah this a product of not yet having a format_urn and wanting contact.urns.tel to resolve to telephone number - def agree this should be the whole thing.